internal/command/command.go

```diff
@@ -128,6 +140,7 @@ func (c Command) AddToParent(parent *cobra.Command) {
 		} else {
 			panic("command implementation needs to provide run functionality")
 		}
+		c.handleUserTracking()
```

Comment:
I wouldn't put the call on this line, I would put it either at the very bottom or at the top. This is in between command and error handling.

Reply:
addressed